config/terraform/application/application.tf

```diff
@@ -55,6 +55,11 @@ module "web_application" {
   max_memory = var.webapp_memory_max
 
   enable_logit = var.enable_logit
+
+  # Uncomment this when we want traffic to be redirected to the maintenance
+  # page during disaster recovery (i.e., while waiting for a database to be
+  # recreated)
+  # send_traffic_to_maintenance_page = true
```

Comment:
Curious if you considered putting this in a variable and controlling it via environment specific `.tfvars` files? E.g `staging.tfvars`?

That way we could test it out whenever we wanted to and would make accidental activation more difficult.